### DISCUSSION TITLE
[Feature Proposal]: Add "place-items-*" utilities

### DISCUSSION THREAD

**Discussion by TheDutchCoder** [tailwindlabs/tailwindcss#1517] [Ideas]
Since center aligning items in flexbox (or a grid) is pretty common by using `justify-center` and `items-center`, it would be very useful to have a `place-items-*`set of utilities which is the shorthand for both these properties.

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/place-items

**Comment by dangdennis** [comment]
This is a really good idea. Besides being super useful, it'll pave the way to learning css grid as well. I haven't felt the need to use CSS grids lately and forgot, but after using grid with tailwind, it's become the gateway.

**Comment by adamwathan** [comment]
Not opposed to this if we can come up with some sensible ones to include (there's a lot of possible combindations). What exact values would you propose we include?

**Comment by jenstornell** [comment]
@adamwathan My previous suggestion today to just have a class named `center` is probably too simplified as there can be other values as well.

What I think most people really want is still to center content vertically and horizontally.

Maybe this can be a good start?

```css
.place-center {
  place-items: center;
}
```

```html
<div class="place-center">
  I'm centered!
</div>
```

Then you can add more values later on. Just make sure that the class name is future proof, which I think it is.
